validate_config.py

```
@@ -0,0 +1,79 @@
+        print(f"   ✅ {key}: {masked}")
+    else:
+        print(f"   ❌ {key}: NOT SET")
+        all_valid = False
+
+print("\n🤖 LLM Configuration:")
+provider = llm_configs["LLM_PROVIDER"]
+print(f"   Provider: {provider}")
+
+if provider == "gemini":
+    if llm_configs["GEMINI_API_KEY"]:
+        print(f"   ✅ GEMINI_API_KEY: Set")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)